components/klaudsolcms/buttons/AppButtonSm.js

```diff
@@ -1,5 +1,9 @@
-const AppButtonSm = ({icon, title, isDisabled}) => {
-    return <button className="btn_general_sm" disabled={isDisabled}> {icon} {title}  </button>    ;
-}
- 
-export default AppButtonSm;
\ No newline at end of file
+const AppButtonSm = ({ icon, title, isDisabled, ...props }) => {
```

Comment:
If there are no statements between const  `AppButtonSm = ( { icon, title, isDisabled, ...props } ) => { `(curly braces) and `return (`, you can just omit `return (` and use `const AppButtonSm = ( { icon, title, isDisabled, ...props } ) => (` (parentheses) instead.

Additionally, use the correct application of new lines for code readability like so:

```suggestion
const AppButtonSm = ({ icon, title, isDisabled, ...props }) => (
    <button 
      className="btn_general_sm" 
      disabled={isDisabled} 
      {...props}
    > {icon} {title}
    </button>
);
```